sample-queries/sample-queries.json

```diff
@@ -270,7 +270,7 @@
         "humanName": "get a user's presence ",
         "requestUrl": "/beta/users/{user-id}/presence",
         "docLink": "https://docs.microsoft.com/en-us/graph/api/cloudcommunications-getpresencesbyuserid?view=graph-rest-beta&tabs=http",
-        "tip": "This query requires a user id and Presence.Read.All permissions. To find user id, you can run:  GET https://graph.microsoft.com/v1.0/users",
+        "tip": "This query requires a user id and Presence.Read.All permissions. To find a user id, you can run:  GET https://graph.microsoft.com/v1.0/users",
```

Comment:
will localization be affected with this change?

Reply:
https://github.com/LokiLabs/microsoft-graph-devx-content/pull/7#discussion_r660819126